modules/au.com.trgtd.tr.email/src/au/com/trgtd/tr/email/task/FetchEmailThread.java

```diff
@@ -43,12 +43,8 @@ public FetchEmailThread() {
      */
     @Override
     public void run() {
-//      LOG.info("Starting email fetch at: " + new Date());
```

Comment:
Why this deletions?

Reply:
@RadekCap Your two comments indicate the struggle I was going though myself. My first approach was to actually delete the logs but then decided to migrate them in a best shot appraoch. I would like to remove the commented out code, but in the context of a different PR. I did not revert the initial deletion in this file but in the others...

Reply:
I restored it for now